deps/rabbit/src/rabbit_fifo.erl

```diff
@@ -894,10 +894,9 @@ tick(Ts, #?MODULE{cfg = #cfg{name = _Name,
                              resource = QName}} = State) ->
     case is_expired(Ts, State) of
         true ->
-            [{mod_call, rabbit_quorum_queue, spawn_deleter, [QName]}];
+            [{aux, {handle_tick, spawn_deleter, [QName]}}];
```

Comment:
I think we always want to launch this process irrespective if there is a tick process running.

Reply:
Done, I think.